packages/@aws-cdk/cloudformation-diff/lib/format.ts

```diff
@@ -19,6 +20,16 @@ export interface FormatStream extends NodeJS.WritableStream {
   columns?: number;
 }
 
+/**
+ * Resource drift status types from CloudFormation
+ */
+export enum ResourceDriftStatus {
+  MODIFIED = 'MODIFIED',
+  DELETED = 'DELETED',
+  NOT_CHECKED = 'NOT_CHECKED',
+  IN_SYNC = 'IN_SYNC',
+}
```

Comment:
We prefer discriminated string unions for internal types these days. Also the enum seems unused outside this file, so no need to export

```suggestion
type ResourceDriftStatus = 'MODIFIED' | 'DELETED' | 'NOT_CHECKED' | 'IN_SYNC';
```

Reply:
Actually, looking further into it, we can probably just import the `StackResourceDriftStatus` directly from the SDK instead of redefining it here, which might be cleaner still